_books/ion-1-1/src/macros/defining_macros.md

```diff
@@ -0,0 +1,76 @@
+## Defining macros
+
+A macro is defined using a `macro` clause within a [module](../modules.md)'s [`macro_table` clause](../modules.md#macro_table).
+
+### Syntax
+```ion
+(macro name signature template)
+```
```

Comment:
The grammar appears in the `modules` section. Per our offline discussion, it would be nice to make it so grammar snippets could be included wherever it made sense. I'll look into that in a follow-on PR.

IMO, the prose description isn't more helpful--you end up reading more words to arrive at the same shallow/vague understanding of the clause. I considered adding examples, but it didn't seem right to have complete examples before explaining all of the sub-elements of the clause. The added clarity may be worth it, though.